test/dotcom/system_status/groups_test.exs

```diff
@@ -0,0 +1,565 @@
+defmodule Dotcom.SystemStatus.GroupsTest do
+  use ExUnit.Case, async: true
+
+  alias Dotcom.SystemStatus.Groups
+  alias Test.Support.Factories.Alerts.Alert
+  alias Test.Support.Factories.Alerts.InformedEntity
+
+  @heavy_rail_lines ["Blue", "Orange", "Red"]
+  @all_rail_lines ["Blue", "Green", "Orange", "Red"]
+  @green_line_branches ["Green-B", "Green-C", "Green-D", "Green-E"]
```

Comment:
Should be alphabetical in order unless they build off of each other. @all...@green...@heavy